lib/appsignal/config.ex

```diff
@@ -174,18 +178,53 @@ defmodule Appsignal.Config do
 
   def minutely_probes_enabled? do
     case Application.fetch_env(:appsignal, :config) do
-      {:ok, value} -> !!value[:enable_minutely_probes]
+      {:ok, value} -> !!Map.get(value, :enable_minutely_probes, false)
       _ -> false
     end
   end
 
   def error_backend_enabled? do
     case Application.fetch_env(:appsignal, :config) do
-      {:ok, value} -> !!value[:enable_error_backend]
+      {:ok, value} -> !!Map.get(value, :enable_error_backend, true)
```

Comment:
There was a bug here: The docs say it was meant to default to `true`, but if the config is present and this key is not present, then `value[:enable_error_backend]` is `nil`, and `!!nil` is `false`.